pages/verify/[discordServerId]/[discordMemberId]/[customLink].tsx

```diff
@@ -18,6 +23,27 @@ import styles from "../../../../styles/Verify.module.scss";
 type Props = {
   discordServerName: string;
   starknetNetwork: NetworkName;
+  signMessage: (message: any) => Promise<Signature>;
+};
+
+type StarknetAccount = {
```

Comment:
Do not define types here, it's not clean. If you want to define types (other than Props), define them in the folder `types`